content/engineering/cloud/saas/index.md

```diff
@@ -1,41 +1,207 @@
 # Cloud Software-as-a-Service Team
 
+The Cloud Software-as-a-Service (SaaS) Team is responsible for the **service management** part of the Sourchegraph cloud product. The team provides both **customer-facing** and **internal capabilities that enable our customers to use Sourcegraph cloud product as a service**.
```

Comment:
Should the highlight be shorter (say, just internal) to contrast with customer-facing?

Reply:
it might better sense to highlight the `and` to as I wanted to keep this whole sentence as TL;DR;